### PR TITLE
Add information regarding configuring session cleanup based on expiry time

### PR DESCRIPTION
**Pull request by piraveena**
## Purpose
Resolves https://github.com/wso2/product-is/issues/12803
Screen Shot 2022-03-04 at 6 00 55 PM

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/setup/data-purging.md [hunk 1]
```diff
@@ -238,6 +238,13 @@ You can perform data purging by clearing the session data using the script given
     !!! info
 
         For DB2, MySQL, Oracle, and Postgre database scripts, see [Stored Procedures](https://github.com/wso2/carbon-identity-framework/tree/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures).
+   
+        You can change the session cleanup task in the stored procedure according to your DB policies. You can clean the session data either based on the time created or the expired time. By default, the session clean up task cleans the data based on the time session was created.
```
Comment (nilminiwso2) [inline en/docs/setup/data-purging.md:242]:
```suggestion
        You can change the session cleanup task in the stored procedure according to your DB policies. You can clean the session data either based on the session created time or the session expiry time. By default, the session created time is used.
```

Comment (nilminiwso2) [inline en/docs/setup/data-purging.md:242]: How this renders properly in the build.

Comment (nilminiwso2) [reply en/docs/setup/data-purging.md:242]: Is this only applicable to 5.10?

Comment (piraveena) [reply en/docs/setup/data-purging.md:242]: This is applicable to other versions too. This is the reported version. Once this is reviewed and approved, will do the same changes for other versions too.

Comment (piraveena) [reply en/docs/setup/data-purging.md:242]: It renders properly. But there is an issue in the sidebar rendering in the local setup. That should be a issue in the local dev setup
Screen Shot 2022-03-04 at 10 22 06 AM

File: en/docs/setup/data-purging.md [hunk 2]
```diff
@@ -237,9 +237,14 @@ You can perform data purging by clearing the session data using the script given
 
     !!! info
 
-        For DB2, MySQL, Oracle, and Postgre database scripts, see [Stored Procedures](https://github.com/wso2/carbon-identity-framework/tree/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures).
+        - For DB2, MySQL, Oracle, and Postgre database scripts, see [Stored Procedures](https://github.com/wso2/carbon-identity-framework/tree/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures).
+        - You can change the session cleanup task in the stored procedure according to your DB policies. You can change the session cleanup task in the stored procedure according to your DB policies. You can clean the session data either based on the session created time or the session expiry time. By default, the session created time is used.
+             ```
```
Comment (AnuradhaSK) [inline en/docs/setup/data-purging.md:242]: Shall we mention this example is to EXPIRY_TIME based session cleanup?
When I saw the code blog after the above point, I felt like the given code snippet is for default case

Comment (piraveena) [reply en/docs/setup/data-purging.md:244]: fixed

**Review by AnuradhaSK — APPROVED** [review]

File: en/docs/setup/data-purging.md [hunk 3]
```diff
@@ -237,9 +237,16 @@ You can perform data purging by clearing the session data using the script given
 
     !!! info
 
-        For DB2, MySQL, Oracle, and Postgre database scripts, see [Stored Procedures](https://github.com/wso2/carbon-identity-framework/tree/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures).
+        - For DB2, MySQL, Oracle, and Postgre database scripts, see [Stored Procedures](https://github.com/wso2/carbon-identity-framework/tree/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures).
+        - You can change the session cleanup task in the stored procedure according to your DB policies. You can change the session cleanup task in the stored procedure according to your DB policies. You can clean the session data either based on the session created time or the session expiry time. By default, the session created time is used.
```
Comment (nilminiwso2) [inline en/docs/setup/data-purging.md:241]:
```suggestion
        - You can change the session cleanup task in the stored procedure according to your DB policies. You can clean the session data either based on the session created time or the session expiry time. By default, the session created time is used.
```

File: en/docs/setup/data-purging.md [hunk 4]
```diff
@@ -237,9 +237,16 @@ You can perform data purging by clearing the session data using the script given
...
+        - A sample script is given below where you can clear the session data based on the EXPIRY_TIME
```
Comment (nilminiwso2) [inline en/docs/setup/data-purging.md:242]:
```suggestion
        - A sample script is given below, which will clear the session data based on the `EXPIRY_TIME`.
```

**Pull request merged**